src/Place/Place.php

```diff
@@ -147,7 +161,31 @@ public function updateMajorInfo(
         CalendarInterface $calendar,
         Theme $theme = null
     ) {
-        $this->apply(new MajorInfoUpdated($this->actorId, $title, $eventType, $address, $calendar, $theme));
+        $this->apply(
+            new MajorInfoUpdated(
+                $this->placeId,
+                $title,
+                $eventType,
+                $address,
+                $calendar,
+                $theme
+            )
+        );
+    }
+
+    /**
+     * @param Address $address
+     * @param Language $language
+     */
+    public function updateAddress(Address $address, Language $language)
```

Comment:
Eventueel een if statement inbouwen om enkel een event aan te maken als het adres daadwerkelijk is gewijzigd?